docs/pages/en-us/Client-API.md

```diff
@@ -108,28 +109,31 @@ In the [`ConfigurationSource`](#ConfigurationSource) above, we used JSON-encoded
 - `CAF_CONFIG_PATH: /etc/sampleapp/config`
 - `CAF_APPNAME: sampleappgroup/sampleapp`
 
-Given this configuration, you would configure Auditing by creating a file named `cfg_sampleappgroup_sampleapp_KafkaAuditConfiguration` in the `/etc/sampleapp/config/` directory. The contents of this file should be similar to the following:
+Given this configuration, you would configure Auditing by creating a file named `cfg_sampleappgroup_sampleapp_ElasticsearchAuditConfiguration` in the `/etc/sampleapp/config/` directory. The contents of this file should be similar to the following:
 
 	{
-	    "bootstrapServers": "<kafka broker>:<port number>",
-	    "acks": "all",
-	    "retries": "0"
+	    "hostAndPort": "<Elasticsearch_Cluser_Node_1>:<Port_Number>,<Elasticsearch_Cluser_Node_2>:<Port_Number>",
+	    "clusterName": "elasticsearchcluster",
+	    "numberOfShards": "5",
+	    "numberOfReplicas": "1"
 	}
 
-
 where:
-* `bootstrapServers` refers to one or more of the nodes of the Kafka cluster as a comma-separated list.
-* `acks` is the number of nodes in the cluster which must acknowledge an audit event when it is sent.
+
+- `hostAndPort` refers to one or more of the nodes of the Elasticsearch cluster as a comma-separated list.
+- `clusterName` name of the Elasticsearch cluster. Defaults to "elasticsearch".
+- `numberOfShards` the number of primary shards that an index should have. Defaults to 5.
+- `numberOfReplicas` the number of replica shards (copies) that each primary shard should have. Defaults to 1.
 
 ### AuditConnection
 
-The `AuditConnection` object represents a logical connection to the persistent storage (that is, Kafka in this implementation). It is a thread-safe object. ***You should expect this object to take some time to construct. The application should hold on to it and re-use it, rather than constantly re-construct it.***
+The `AuditConnection` object represents a logical connection to the persistent storage (that is, Elasticsearch in this implementation). It is a thread-safe object. ***You should expect this object to take some time to construct. The application should hold on to it and re-use it, rather than constantly re-construct it.***
```

Comment:
Just because I know we changed the Getting Started page.  
Replace, "the persistent storage"  
With, "the datastore"